### DISCUSSION TITLE
I started a Minecraft Server and added the GP plug in but I don't have admin inside. Help

### DISCUSSION THREAD

**Discussion by LordDraven** [GriefPrevention/GriefPrevention#1960] [Q&A]
I started the server and added my account as owner of the server but I don't have Admin permissions. I cannot use any command for admins. What do I have to do to give my account Admin permissions and others. Is this something I hace to change on the config.yml or something else? I have my server in Seekahost.

**Comment by CaitieThompson** [comment]
Just wondering if you ever figured out a solution as I am also experiencing this issue and I'm the owner of the server. My gamertag on the console is set to admin but it doesnt go into effect in game..

**Comment by LordDraven** [comment]
Greetings,

Yes,  I found a solution. On your server page go to console and run the command to grant admin permission from there. Tbh I don’t remember the command right now but you can find it on the faq of the GP plugin. Remember that this needs to be done outside the game on your Server console.

Ricardo Cortez
